### DISCUSSION TITLE
Visualization of  RBC  fails in Paraview

### DISCUSSION THREAD

**Discussion by nbdudu** [UvaCsl/HemoCell#110] [General discussion]
Hello,
I simulated the case: Stretchcell and tried to visualize fluid and RBC following the steps in the user guide (use [batchPostProcess.sh] to get the RBC.xmf and load it into Paraview for visualization, select the XDMF reader to import the file.) 

After that, RBC was not displayed. May I ask what went wrong or what operation needs to be done in Paraview?

Thanks!

load RBC.xmf

![paraview_openerror_import](https://github.com/UvaCsl/HemoCell/assets/145785978/a4209fb1-4cfa-4d20-a359-3a2b2b34c1cd)
![paraview_opendata](https://github.com/UvaCsl/HemoCell/assets/145785978/11f001a8-0737-433f-b271-170091aaa0ac)

RBC is not shown in the window:

![paraview_openerror](https://github.com/UvaCsl/HemoCell/assets/145785978/2b7463e8-2845-49ec-b736-13962fd52497)

**Comment by mojtabaAmiraslanpour** [comment]
Hi,

If you are using pre-inlet, there won't be any RBCs at the beginning of the simulation inside the main domain i.e. RBCxxx files.
If that is not the case you can use the "Merge Blocks" feature to see if it solves the problem. There is a small bug in writing the XML files which causes that.

This being said you have to make sure that you actually have some cells in the domain. You can check this when you are running the case and report the number of cells in the domain.

**Reply by nbdudu** [reply]
Hi,
Thanks for your suggestion! I will check it out. 

Since the cell deformation in the output  file (stretch.log) is relatively consistent with the result in the paper (Cellular ...,2017,Fig.4), can it be considered that there is a cell in the flow field?

**Reply by mojtabaAmiraslanpour** [reply]
Yes, if you see the stretch results then the cell is there.
